lib/membrane/compositor_element.ex

```diff
@@ -143,7 +187,11 @@ defmodule Membrane.VideoCompositor.CompositorElement do
     state = %{state | internal_state: internal_state}
 
     if all_input_pads_received_end_of_stream?(context.pads) do
-      {{:ok, end_of_stream: :output}, state}
+      if state.live do
+        {{:ok, end_of_stream: :output, stop_timer: :render_frame}, state}
+      else
+        {{:ok, end_of_stream: :output}, state}
+      end
```

Comment:
```suggestion
     actions = if all_input_pads_received_end_of_stream?(context.pads) do
      stop = if state.live, do: [stop_timer: :render_frame], else: []
      [end_of_stream: :output] ++ stop
    else
      []
    end

    {{:ok, actions}, state}
```
Obviously it is also for the line above and below, but GitHub don't let do this proper way.